src/OnboardingSPA/components/HeadingWithSubHeading/SiteGen/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Interface Cards with standard design.
+ *
+ * @param {Object} root0
+ * @param {string} root0.title
+ * @param {string} root0.subtitle
+ * @param {Object} root0.brandName
+ */
+const HeadingWithSubHeading = ( { title, subtitle, brandName } ) => {
```

Comment:
Can't this component itself be `brandName` aware? why does that need to be passed?